google/cloud/pubsub/internal/tracing_message_callback_test.cc

```diff
@@ -46,42 +47,85 @@ using ::google::cloud::testing_util::SpanKindIsInternal;
 using ::google::cloud::testing_util::SpanNamed;
 using ::google::cloud::testing_util::SpanWithParent;
 using ::testing::AllOf;
+using ::testing::ByMove;
 using ::testing::Contains;
+using ::testing::Return;
 
 pubsub::Subscription TestSubscription() {
   return pubsub::Subscription("test-project", "test-sub");
 }
 
 std::shared_ptr<MessageCallback> MakeTestMessageCallback(
-    std::shared_ptr<MessageCallback> mock) {
+    std::shared_ptr<MessageCallback> mock, Options opts) {
```

Comment:
what was the point of this? we only pass it `Options{}` at the moment.

Reply:
Removed.